cli/Cargo.toml

```diff
@@ -45,7 +45,7 @@ tokio-rustls = { workspace = true }
 tracing = { workspace = true }
 tracing-futures = { workspace = true }
 tracing-subscriber = { version = "^0.3.16", features = ["env-filter", "fmt"] }
-viceroy-lib = { path = "../lib", version = "^0.10.2" }
+viceroy-lib = { path = "../lib", version = "=0.10.3" }
```

Comment:
Note the change from `^` to `=` here. With an intra-repo dependency like this we want to use only this exact version of the other crate.